docs/HowTo/Get-Started/Run-Docker-Image.md

```diff
@@ -88,32 +88,34 @@ services:
 
   besu_node:
     image: hyperledger/besu:latest
-    command: ["--genesis-file=/opt/besu/data/depositContractGenesis.json",
+    command: ["--network=goerli",
               "--data-path=/opt/besu/data/data",
               "--host-allowlist=*",
+              "--sync-mode=FAST",
               "--rpc-http-enabled",
               "--rpc-http-cors-origins=*",
-              "--rpc-http-api=ETH,NET,CLIQUE,DEBUG,MINER,NET,PERM,ADMIN,EEA,TXPOOL,PRIV,WEB3",
-              "--miner-enabled=true",
-              "--miner-coinbase=0xfe3b557e8fb62b89f4916b721be55ceb828dbd73",
-              "--min-gas-price=0"]
+              "--rpc-http-api=ETH,NET,CLIQUE,DEBUG,MINER,NET,PERM,ADMIN,EEA,TXPOOL,PRIV,WEB3"]
     volumes:
       - ./besu:/opt/besu/data
     ports:
+      # Map the p2p port(30303) and RPC HTTP port(8545) 
       - "8545:8545"
       - "30303:30303"
 
   teku_node:
-    image: pegasyseng/teku:develop
-    command: ["--eth1-deposit-contract-address=dddddddddddddddddddddddddddddddddddddddd",
+    image: pegasyseng/teku:latest
```

Comment:
Are we intentionally changing to the `latest` tag here?  All of the other docker commands are using `develop` and we've generally been encouraging people to run from `develop` to get all of the latest fixes.

Reply:
You're right. I forgot we recommend using develop for Teku.